### ISSUE TITLE
Display error message when hurl input can not be decoded

### ISSUE THREAD

**Issue by fabricereix** [Orange-OpenSource/hurl#139]
```
$ cat /usr/bin/ls | hurl
thread 'main' panicked at 'Something went wrong reading standard input: Custom { kind: InvalidData, error: "stream did not contain valid UTF-8" }', packages/hurl/src/main.rs:665:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by fabricereix** [comment]
Here is the new error message

```
$ cat /usr/bin/ls | hurl
error: Input stream can not be read - stream did not contain valid UTF-8
```

**Comment by fabricereix** [comment]
fix in #140
